schematic/models/validate_attribute.py

```diff
@@ -336,16 +340,16 @@ def generate_content_error(
         """
 
         error_row = row_num
-        error_val = iterable_to_str_list(set(invalid_entry)) if invalid_entry else None
+        #error_val = iterable_to_str_list(set(invalid_entry)) if invalid_entry else None
 
         # log warning or error message
         if val_rule.startswith("recommended"):
             error_message = f"Column {attribute_name} is recommended but empty."
             error_row = None
-            error_val = None
+            #error_val = None
```

Comment:
Are these supposed to be commented out?

Reply:
@andrewelamb, this PR is still in progress and not ready for review